layouts/fastly/helix.vcl

```diff
@@ -1761,6 +1910,19 @@ sub hlx_bereq {
       } else {
         set bereq.http.hlx-forwarded-host = req.http.X-Orig-Host;
       }
+    } elsif (req.backend == F_UniversalRuntime) {
+      // Set the hostame for the universal Runtime service
+      set bereq.http.Host = req.http.X-Backend-Host;
```

Comment:
Use the backend hostname set here https://github.com/adobe/helix-publish/pull/704/files#diff-9e59debf5608a9bdb1f1a87b891b2f995823ea79254a122b8d1fcb2bb1f69785R567 (for instance)

As `*.anywhere.run` will always go to Fastly, setting the subdomain name here allows Fastly to pick the correct gateway service.